src/base/rss/private/rss_parser.cpp

```diff
@@ -791,5 +791,5 @@ void Parser::addArticle(QVariantHash article)
     }
 
     m_articleIDs.insert(localId.toString());
-    m_result.articles.prepend(article);
+    m_result.articles.append(article);
```

Comment:
not so sure, but I suspect this replacement would be ok.

Reply:
AFAIK, it's intended to use `prepend()` here (it's part of logic). Please don't touch it.

Reply:
reverted